Lib/site.py

```diff
@@ -457,63 +457,58 @@ def gethistoryfile():
         '.python_history')
 
 
-def enablerlcompleter():
-    """Enable default readline configuration on interactive prompts, by
-    registering a sys.__interactivehook__.
+def register_readline():
+    """Enable default readline configuration on interactive prompts.
```

Comment:
Technically, this function *is* the "interactive hook". The docstring should probably clarify that.

Also, even though it isn't documented, I think we should maintain a function named `enablerlcompleter()` which, when calls, does
```py
sys.__interactivehook__ = register_readline
```
It's likely that someone, somewhere is relying on that function existing, and it's only 2-3 lines to keep them happy.

Reply:
I added `enablerlcompleter()` back and changed the wording in the docstrings. Should I add `enablerlcompleter()` and `register_readline()` to the documentation?